src/Libraries/RevitNodes/Elements/ImportInstance.cs

```diff
@@ -95,6 +106,46 @@ private void InitImportInstance(string satPath, XYZ translation = null)
             ElementBinder.SetElementForTrace(importInstance);
         }
 
+        /// <summary>
+        /// Initialize an ImportInstance element
+        /// </summary>
+        /// <param name="satPath"></param>
+        /// <param name="translation"></param>
+        /// <param name="view"></param>
+        private void InitImportInstance(string satPath, Revit.Elements.Views.View view, XYZ translation = null)
```

Comment:
The code of private void InitImportInstance(string satPath, XYZ translation = null) and private void InitImportInstance(string satPath, Revit.Elements.Views.View view, XYZ translation = null) are similiar, could you simplify them?  InitImportInstance(string satPath, XYZ translation = null) can just call InitImportInstance(string satPath, Revit.Elements.Views.View view, XYZ translation = null) with a specific view parameter.
You can also simplify of the constructor of ImportInstance.